pkg/cluster/cluster.go

```diff
@@ -312,7 +312,7 @@ func (c *Cluster) createMachineRunArgs(machine *Machine, name string, i int) []s
 			"-v", "/sys/fs/cgroup:/sys/fs/cgroup:rw")
 
 	} else {
-		runArgs = append(runArgs, "-v", "/sys/fs/cgroup:/sys/fs/cgroup:ro")
```

Comment:
Non-recursive read-only mounts don't seem to be supported by the CLI, although they seem to be a thing if using the API directly.